<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGridDialog</class>
 <widget class="QDialog" name="NewGridDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Grid</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>14</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="rbEmptyGrid">
          <property name="text">
           <string>New empty grid</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbStandardGrid">
          <property name="text">
           <string>New standard grid</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <property name="verticalSpacing">
         <number>9</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item row="2" column="1">
         <widget class="QFrame" name="frame">
          <property name="styleSheet">
           <string notr="true">.QFrame{border: 0px solid black; border-left: 1px solid grey}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="verticalSpacing">
            <number>6</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Origin</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="verticalSpacing">
              <number>3</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_6">
               <property name="minimumSize">
                <size>
                 <width>21</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>x:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="spOx">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="suffix">
                <string>mm</string>
               </property>
               <property name="maximum">
                <double>99990.990000000005239</double>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_7">
               <property name="minimumSize">
                <size>
                 <width>21</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>y:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="spOy">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="suffix">
                <string>mm</string>
               </property>
               <property name="maximum">
                <double>99990.990000000005239</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Delta</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <layout class="QGridLayout" name="gridLayout">
             <property name="verticalSpacing">
              <number>3</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>dx:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="spDx">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="suffix">
                <string>mm</string>
               </property>
               <property name="maximum">
                <double>99990.990000000005239</double>
               </property>
               <property name="value">
                <double>50.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>dy:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="spDy">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="suffix">
                <string>mm</string>
               </property>
               <property name="maximum">
                <double>99990.990000000005239</double>
               </property>
               <property name="value">
                <double>50.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0" colspan="5">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>19</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Number of connections:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbNbConnections">
               <property name="currentIndex">
                <number>2</number>
               </property>
               <item>
                <property name="text">
                 <string>No Connections</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Four Connetions</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Height Connections</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="4">
            <layout class="QGridLayout" name="gridLayout_4">
             <property name="verticalSpacing">
              <number>3</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>x:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="spEx">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="suffix">
                <string>mm</string>
               </property>
               <property name="maximum">
                <double>99990.990000000005239</double>
               </property>
               <property name="value">
                <double>2000.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>y:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="spEy">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="suffix">
                <string>mm</string>
               </property>
               <property name="maximum">
                <double>99990.990000000005239</double>
               </property>
               <property name="value">
                <double>3000.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>End</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>8</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>136</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rbStandardGrid</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>NewGridDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>NewGridDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>30</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
